components/dashboard/src/workspaces/StartWorkspaceModal.tsx

```diff
@@ -35,23 +35,23 @@ export function StartWorkspaceModal(p: StartWorkspaceModalProps) {
     useEffect(() => setSelection(computeSelection()), [p.recent, p.selected]);
 
     const list = (selection === 'Recent' ? p.recent : p.examples).map(e =>
-        <a key={e.title} href={e.startUrl} className="rounded-xl group hover:bg-gray-100 flex p-4 my-1">
+        <a key={e.title} href={e.startUrl} className="rounded-xl group hover:bg-gray-100 dark:hover:bg-gray-800 flex p-4 my-1">
             <div className="w-full">
-                <p className="text-base text-gray-800 font-semibold">{e.title}</p>
+                <p className="text-base text-gray-800 dark:text-gray-200 font-semibold">{e.title}</p>
                 <p>{e.description}</p>
             </div>
         </a>);
 
     return <Modal onClose={p.onClose} visible={p.visible}>
         <h3 className="pb-2">New Workspace</h3>
```

Comment:
**issue**: Reminder to fix [this empty list](https://github.com/gitpod-io/gitpod/pull/3901/files#diff-6dfa3291e87a2551003c765d91b740569556490925a3d755ba690e17e99e6546R72) when using only the GitLab provider. Mirroring our examples to our GitLab group could work. However ,this is out of the scope  of this PR, see https://github.com/gitpod-io/gitpod/issues/3531.